addon/serializers/json-api.js

```diff
@@ -24,30 +24,32 @@ const dasherize = Ember.String.dasherize;
 
   This serializer normalizes a JSON API payload that looks like:
 
-  ```js
+  ```app/models/player.js
+  import DS from 'ember-data';
 
-    // models/player.js
-    import DS from "ember-data";
+  const { attr, belongsTo } = DS;
```

Comment:
Seems inconsistent, since they are used in other examples. Should those be removed?